Kalyani/HTML/Assignment4/KalyaniA4_1.html

```diff
@@ -1,54 +1,50 @@
 <!DOCTYPE html>
 <html lang="en">
-
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Assignment4_1</title>
-    <style>
-        table,
-        th,
-        td {
-            border: 1px solid black;
-            border-collapse: collapse;
-            padding: 15px;
-            background-color: seagreen;
-            width: auto;
-        }
-        
-        table.center {
-            margin-left: 5px;
-            margin-right: 5px;
-        }
-    </style>
-</head>
-
-<body>
-    <table class="center">
-        <caption><i>A Test table With Merged Cell</i></caption>
-        <tr>
-            <th rowspan="2"></th>
-            <th colspan="2">Average</th>
-            <th rowspan="2">Red Eyes</th>
-        </tr>
-        <tr>
-            <th>height</th>
-            <th>Weight</th>
-        </tr>
-        <tr>
-            <th>Male</th>
-            <td>1.9</td>
-            <td>0.03</td>
-            <td>40%</td>
-        </tr>
-        <tr>
-            <th>Female</th>
-            <td>1.7</th>
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Assignment4_1</title>
+        <style>
+            table,
```

Comment:
dont split table th td in different lines write it as like table, th, td {style u want to apply}.